site/content/en/docs/Reference/fleetautoscaler.md

```diff
@@ -79,6 +79,63 @@ spec:
       seconds: 30
 ```
 
+{{% feature publishVersion="1.37.0" %}}
+Counter-based `FleetAutoscaler` specification below and in the {{< ghlink href="examples/counterfleetautoscaler.yaml" >}}example folder{{< /ghlink >}}:
+
+```yaml
+apiVersion: autoscaling.agones.dev/v1
+kind: FleetAutoscaler
+metadata:
+  name: fleet-autoscaler-counter
+spec:
+  fleetName: fleet-example
+  policy:
+    type: Counter  # Counter based autoscaling
+    counter:
+      # Key is the name of the Counter. Required field.
+      key: players
+      # BufferSize is the size of a buffer of counted items that are available in the Fleet (available capacity).
+      # Value can be an absolute number (ex: 5) or a percentage of desired gs instances (ex: 5%). 
+      # An absolute number is calculated from percentage by rounding up. Must be bigger than 0. Required field.
+      bufferSize: 5
+      # MinCapacity is the minimum aggregate Counter total capacity across the fleet.
+      # If zero, MinCapacity is ignored. 
+      # If non zero, MinCapacity must be smaller than MaxCapacity and bigger than BufferSize.
+      minCapacity: 10
+      # MaxCapacity is the maximum aggregate Counter total capacity across the fleet.
+      # MaxCapacity must be bigger than both MinCapacity and BufferSize. Required field.
+      maxCapacity: 100
+```
+
+List-based `FleetAutoscaler` specification below and in the {{< ghlink href="examples/listfleetautoscaler.yaml" >}}example folder{{< /ghlink >}}:
+
+```yaml
+apiVersion: autoscaling.agones.dev/v1
+kind: FleetAutoscaler
+metadata:
+  name: fleet-autoscaler-list
+spec:
+  fleetName: fleet-example
+  policy:
+    type: List  # List based autoscaling.
+    list:
+      # Key is the name of the List. Required field.
+      key: rooms
+      # BufferSize is the size of a buffer based on the List capacity that is available over the current
+      # aggregate List length in the Fleet (available capacity).
+      # It can be specified either as an absolute value (i.e. 5) or percentage format (i.e. 5%).
+      # Must be bigger than 0. Required field.
+      bufferSize: 5
+      # MinCapacity is the minimum aggregate List total capacity across the fleet.
+      # If zero, it is ignored.
+      # If non zero, it must be smaller than MaxCapacity and bigger than BufferSize.
+      minCapacity: 10
+      # MaxCapacity is the maximum aggregate List total capacity across the fleet.
+      # MaxCapacity must be bigger than both MinCapacity and BufferSize. Required field.
+      maxCapacity: 100
+```
+{{% /feature %}}
+
```

Comment:
Same as you did in fleet.md - need the fields desribed in the section from 144 onwards.

Reply:
same doubt, the fields in the counter and list for the autoscaling to be included in the `spec` field?